.circleci/config.yml

```diff
@@ -64,6 +64,21 @@ jobs:
             CGO_ENABLED=0 equinox release --version=${CIRCLE_SHA1:0:8} --platforms="darwin_amd64 linux_amd64 darwin_386 linux_386 windows_amd64 windows_386" --signing-key=equinox.key --app=$EQUINOX_APP_ID --token=$EQUINOX_ACC_TOKEN --channel dev -- -ldflags "-X main.releaseChannel=dev -X main.build=${CIRCLE_SHA1:0:8}" github.com/rainforestapp/rainforest-cli
             rm -f equinox.key
 
+  release:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Release via GoReleaser
+          command: |
+            export GITHUB_TOKEN=$GITHUB_ACCESS_TOKEN
+            curl -sL https://git.io/goreleaser | bash
```

Comment:
This seems terrible, but I see it's their recommend way of doing things...
https://goreleaser.com/ci/circle/

They have a docker image, would it be better to use that? If only so Circle might have a cached version of the image?

Reply:
Their Docker image doesn't use the same version of Go as we do (it's on the latest 1.16 while we're still on 1.12). It _should_ be fine since all Go versions have been backwards compatible (and really we should just update this repo to use 1.16), but I wanted to minimize changes in this PR.